src/core/src/energy/global/ewald.rs

```diff
@@ -704,6 +704,10 @@ impl Ewald {
 }
 
 impl GlobalPotential for Ewald {
+    fn get_cutoff(&self) -> Option<f64> {
```

Comment:
Can we name this function `cutoff`? The usual way in Rust is to have getter called `foo` and setters called `set_foo`. Here is the corresponding [RFC](https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#gettersetter-apis).